js/project-page-functions.js

```diff
@@ -146,22 +146,35 @@ function processBuildLogJson(myLog, $downloadMenuButton, $buildStatusIcon, $last
     saveDownloadLink(myLog);
     setDownloadButtonState($downloadMenuButton);
     updateTextForDownloadItem(myLog.input_format);
+    updateConversionStatusOnPage($buildStatusIcon, myLog);
+    $revisions.empty();
+}
+
+function updateConversionStatusOnPage($buildStatusIcon, myLog) {
+    if(CONVERSION_TIMED_OUT) {
+        myLog.status = "failed";
+        if(!myLog.errors) {
+            myLog.errors = [];
+        }
+        const errorMsg = "Conversion Timed Out!\nStarted " + timeSince(new Date(myLog.created_at)) + " ago";
```

Comment:
I'd think you'd want to go by started_at, even though they are usually very close.